src/xcode/ENA/ENA/Source/Services/HealthCertificate/HealthCertificateService.swift

```diff
@@ -829,6 +829,8 @@ class HealthCertificateService: HealthCertificateServiceServable {
 				person.mostRecentWalletInfoUpdateFailed = true
 				completion?()
 			}
+			
+			self.updateGradients()
```

Comment:
Do we need this?

Reply:
Yes, after update the `dccWalletInfo`, wo have to trigger a recalculation of the `GradientType` in the `HealthCertificateService`.